vNext/src/NuGet/Microsoft.Orleans.Core.nuspec

```diff
@@ -0,0 +1,50 @@
+﻿<?xml version="1.0"?>
+<package xmlns="http://schemas.microsoft.com/packaging/2010/07/nuspec.xsd">
+  <metadata>
+    <id>Microsoft.Orleans.Core</id>
+    <version>$version$</version>
+    <title>Microsoft Orleans Core Library</title>
+    <authors>Microsoft</authors>
+    <owners>Microsoft,Orleans</owners>
+    <projectUrl>https://github.com/dotnet/Orleans</projectUrl>
+    <licenseUrl>https://github.com/dotnet/Orleans#license</licenseUrl>
+    <iconUrl>https://raw.githubusercontent.com/dotnet/orleans/gh-pages/assets/logo_128.png</iconUrl>
+    <summary>
+      Core library of Microsoft Orleans - Orleans.dll
```

Comment:
I think we should use a namespaced dll name and not include it in the package description